classes/Http/Controller/Admin/TalksController.php

```diff
@@ -74,6 +74,10 @@ public function viewAction(Request $req)
 
     public function rateAction(Request $req)
     {
+        if (!is_numeric($req->get('rating'))) {
```

Comment:
@BackEndTea 

I haven't looked into how filtering and validation is currently managed within the project, but if there's none, what do you think about using appropriate components sooner or later?

Reply:
The ```BaseController``` has a ```validate``` function, Which is used exactly once, in the ```processAction``` of the ```SignupController```.

Reply:
I was thinking more along the lines of creating specific validators for each use case.

Reply:
As in, some kind of abstract validation class/interface, and in this case a ```RatingValidation``` which checks if the request object fills the pre-defined criteria?

Reply:
Well, not exactly the entire request object, but the relevant request data?

Reply:
I wouldn't mind that, but its outside of the scope for this PR